pkg/webui/console/views/application/index.js

```diff
@@ -169,17 +174,26 @@ export default class Application extends React.Component {
     }
 
     return (
-      <Switch>
-        <Route exact path={`${match.path}`} component={ApplicationOverview} />
-        <Route path={`${match.path}/general-settings`} component={ApplicationGeneralSettings} />
-        <Route path={`${match.path}/api-keys`} component={ApplicationApiKeys} />
-        <Route path={`${match.path}/link`} component={ApplicationLink} />
-        <Route path={`${match.path}/devices`} component={Devices} />
-        <Route path={`${match.path}/collaborators`} component={ApplicationCollaborators} />
-        <Route path={`${match.path}/data`} component={ApplicationData} />
-        <Route path={`${match.path}/payload-formatters`} component={ApplicationPayloadFormatters} />
-        <Route path={`${match.path}/integrations`} component={ApplicationIntegrations} />
-      </Switch>
+      <React.Fragment>
+        <IntlHelmet
+          titleTemplate={m.title}
+          values={{
+            appName: application.name || appId,
+            siteName: window.SITE_NAME,
```

Comment:
To obtain the environment values, use the [`EnvProvider`](https://github.com/TheThingsNetwork/lorawan-stack/blob/master/pkg/webui/lib/components/env.js#L52) context provider. You can then get the values from the component's props.
You'll have to add the new values [here](https://github.com/TheThingsNetwork/lorawan-stack/blob/master/pkg/webui/console.js#L32).